packages/backend/src/config/discoveryConfig.ts

```diff
@@ -42,6 +49,7 @@ function createConfigFromTemplate(
           },
         },
       },
+      // Maybe we should remove that if there is no multisig support?
       'LayerZero Multisig': {
         ignoreInWatchMode: ['nonce'],
       },
```

Comment:
Removing the override is not necessary as Discovery does not check if all the overrides were used.